Cargo.toml

```diff
@@ -7,17 +7,17 @@ description = "Open Telemetry fox Axum and Tracing"
 
 [dependencies]
 http = "1.1.0"
-opentelemetry = { version = "0.23", default-features = false, features = [
+opentelemetry = { version = "0.24", default-features = false, features = [
```

Comment:
[tracing-opentelemetry-instrumentation-sdk](https://github.com/davidB/tracing-opentelemetry-instrumentation-sdk) doesn't support `opentelemetry@0.24` and `tracing-opentelemetry@0.25` yet